client/src/js/modules/runs/views/overview.js

```diff
@@ -0,0 +1,53 @@
+define([
+    'marionette',
+
+    'collections/bls',
+    
+    'modules/stats/views/beamline',
+    
+    'views/filter',
+    'templates/runs/overview.html',
+    'jquery'
+], function(Marionette, Beamlines, BeamlineStatsView, FilterView, template, $) {
+    return Marionette.LayoutView.extend({
+        template: template,
+        className: 'content',
+        regions: {
+            pills: '.beamline-pills',
+            stats: '.beamline-stats'
+        },
+        showBeamlineStats: function(beamline) {
+            var regexExp = new RegExp(/\/bl\/.*/)
+            var url = window.location.pathname
+            if (beamline) {
+                url = url.replace(regexExp, '')+'/bl/'+ beamline
+                window.history.pushState({}, '', url)
+                this.stats.show(new BeamlineStatsView({ bl: beamline }))
+            } else {
+                url = url.replace(regexExp, '')
+                window.history.pushState({}, '', url)
+                this.stats.currentView.destroy()
+            }
+        },
+        showFilter: function() {
+            this.beamlinePills = new FilterView({
+                url: false,
+                collection: null,
+                name: 'bl',
+                filters: this.beamlines.map(function(beamline) {
+                    return { id: beamline.get('BEAMLINE'), name: beamline.get('BEAMLINE') }
+                })
+            })
+        
+            this.listenTo(this.beamlinePills, 'selected:change', this.showBeamlineStats.bind(this))
+            this.pills.show(this.beamlinePills)
+        },
+        initialize: function() {
```

Comment:
It would be nice if this also took into accound the url when it loads.
For example if someone types in /runs/overview/bl/i03 it should load the stats for i03. Similar to how the stats/bl/i03 page works.